index.html

```diff
@@ -3,10 +3,70 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="index.js"></script>
+    <style type="text/css"></style>
     <title>My To-Do</title>
+    <style>
+
+      .header{
+        display: flex;
+        justify-content: center;
+      }
+
+      .container-1{
+        display: flex;
+        flex-direction: column;
+      }
+
+      .container-1 div{ 
+        border:1px #ccc solid;
+        padding:10px;
+      }
+
+      .box-1{
+        flex: 2;
+      }
+
+      .box-2{
+        flex:1;
+      }
+      .btn{
+        display: flex;
+        justify-content: end;
+        gap: 2px;
+      }
+
+    
+
+    </style>
   </head>
   <body>
-    <h1>Hellow World</h1>
-    <script src="index.js"></script>
+
+    <section>
+      <div class ='header'>
+        <h1>My To-Do List</h1>
+      </div>
+      <div class ='container-1'>
+
+      <div class ='box-1'>
+        <h3>Input</h3>
+        <div class ='btn'>
+        <Button class = 'b1'>Add</Button>
+      </div>
+      </div>
+
+      <div class='Box 2'>
+        <h3>List</h3>
+        <div class = 'btn'>
```

Comment:
Hard to tell what the intent is for this div container. IMO, we're too early in the development for adding the edit and delete buttons.  I think you will realize how you want to use the edit and delete buttons as you start building out the features. Feel free to leave them for now.